packages/block-library/src/navigation-link/index.php

```diff
@@ -344,3 +344,26 @@ function register_block_core_navigation_link() {
 	);
 }
 add_action( 'init', 'register_block_core_navigation_link' );
+
+/**
+ * Disables the display of tabs for the Navigation Link block.
+ *
+ * @param array $settings Default editor settings.
+ * @return array Filtered editor settings.
+ */
+function gutenberg_disable_tabs_for_navigation_link_block( $settings ) {
+	$current_tab_settings = _wp_array_get(
+		$settings,
+		array( '__experimentalBlockInspectorTabs' ),
+		array()
+	);
+
+	$settings['__experimentalBlockInspectorTabs'] = array_merge(
+		$current_tab_settings,
+		array( 'core/navigation-link' => false )
```

Comment:
One thing is that this should probably also be applied to `core/navigation-submenu`, the other main block type in the navigation list view. (cc @getdave @scruffian @draganescu @MaggieCabrera for thoughts on this)

Reply:
Actually, just testing and I notice submenu already doesn't have any tabs (it has no appearance tools).

It might still be worth disabling tabs though, as that's not a reliable situation (plugins can modify things, and the situation might change in the future).

Reply:
+1 to disabling on navigation-submenu

Reply:
✅ Done: https://github.com/WordPress/gutenberg/pull/46321/commits/7d4e31fa9964a1445385c729b0563945342b259a